api/v1alpha1/diskchaos_types.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:root=true
+// +kubebuilder:printcolumn:name="duration",type=string,JSONPath=`.spec.duration`
+// +chaos-mesh:experiment
+
+// DiskChaos is the Schema for the diskchaos API
+type DiskChaos struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec defines the behavior of a disk chaos experiment
+	Spec DiskChaosSpec `json:"spec"`
+
+	// +optional
+	// Most recently observed status of the disk chaos experiment
+	Status DiskChaosStatus `json:"status"`
+}
+
+var _ InnerObjectWithSelector = (*DiskChaos)(nil)
+var _ InnerObject = (*DiskChaos)(nil)
+
+type DiskChaosAction string
+
+const (
+	Fill   DiskChaosAction = "fill"
```

Comment:
how about renaming it, so it has the same style as others?

Reply:
I agree with you.

Reply:
Fixed.